pkg/volume/glusterfs/glusterfs.go

```diff
@@ -78,11 +78,12 @@ const (
 	// default values, but they play a different role and
 	// could take a different value. Only thing we need is:
 	// absGidMin <= defGidMin <= defGidMax <= absGidMax
-	absoluteGidMin = 2000
-	absoluteGidMax = math.MaxInt32
-	heketiAnn      = "heketi-dynamic-provisioner"
-	glusterTypeAnn = "gluster.org/type"
-	glusterDescAnn = "Gluster: Dynamically provisioned PV"
+	absoluteGidMin          = 2000
+	absoluteGidMax          = math.MaxInt32
+	heketiAnn               = "heketi-dynamic-provisioner"
+	glusterTypeAnn          = "gluster.org/type"
+	glusterDescAnn          = "Gluster: Dynamically provisioned PV"
+	linuxGlusterMountBinary = "mount.glusterfs"
```

Comment:
@jsafrane Yes, there is one and while following the PR which introduced, I got an impression that the `path` mentioned with `gciGlusterMountBinary` is very much specific to GCI.  so, if I depend on that path, it may cause problems in future.  I am also trying to put that path as `gci-linux` specific via PR https://github.com/kubernetes/kubernetes/pull/45938 .